dist/images/daemonset.sh

```diff
@@ -157,6 +161,8 @@ ovn_nb_raft_election_timer=${OVN_NB_RAFT_ELECTION_TIMER:-1000}
 echo "ovn_nb_raft_election_timer: ${ovn_nb_raft_election_timer}"
 ovn_sb_raft_election_timer=${OVN_SB_RAFT_ELECTION_TIMER:-1000}
 echo "ovn_sb_raft_election_timer: ${ovn_sb_raft_election_timer}"
+ovn_master_count=${OVN_MASTER_COUNT:-"1"}
+echo "onv_master_count: ${ovn_master_count}"
```

Comment:
typo ("onv")

Reply:
You probably reviewed the files in the first commit, this has already been taken care of in the later commits

Reply:
I'ts like this in the latest commit. Can you squash all these commits please?

Reply:
@trozet @danwinship let me look into this

Reply:
@danwinship, @trozet  my bad,  the typo still exists, I had the same typo on the line above that had failed the ci tests and which I had fixed in an earlier commit, but forgot to take care of this typo....will fix it in the next push